draft-ietf-ppm-dap.md

```diff
@@ -2979,25 +3160,22 @@ ciphersuite indicated by the HPKE configuration.
 
 When the Leader receives a `Query` in the request from the Collector during
 collection initialization ({{collect-init}}), it must first check that the
-batch determined by the query can be collected. It does so as described here.
-The Helper performs the same check when it receives a `BatchSelector` in the
-request from the Leader for its aggregate share ({{collect-aggregate}}).
+batch determined by the query can be collected. The Helper performs the same
+check when it receives a `BatchSelector` in the request from the Leader for its
+aggregate share ({{collect-aggregate}}).
 
 First, the Aggregator checks if the request (the `CollectionJobReq` for the
 Leader and the `AggregateShareReq` for the Helper) identifies a valid set of
 batch buckets ({{batch-buckets}}). If not, it MUST abort the request with
-"batchInvalid".
+`batchInvalid`.
 
-Next, the Aggregator checks that batch contains a valid number of reports. The
-Aggregator checks that `len(X) >= min_batch_size`, where `X` is the set of
-reports successfully aggregated into the batch and `min_batch_size` is the
-minimum batch size for the task. If this check fails, then Helpers MUST abort
-with an error of type "invalidBatchSize". Leaders SHOULD wait for more reports
-to be validated and try the collection job again later.
+Next, the Aggregator checks that the number of reports in the batch is equal to
+or greater than the task's minimum batch size. If not, then the Helper MUST
+abort with error `invalidBatchSize`. The Leader SHOULD wait for more reports to
+be validated and try the collection job again later.
 
-Next, the Aggregator checks if the set of batch buckets identified by the
-request overlaps with the batch buckets that have already been collected. If
-so, it MUST abort with "batchOverlap".
+Next, the Aggregator checks if any of the batch buckets identified by the
+request have already been collected. If so, it MUST abort with "batchOverlap".
```

Comment:
*Batches* (which consist of one or more batch buckets) can overlap but I'm pretty sure *batch buckets* can't.